lib/actions.js

```diff
@@ -1,17 +1,35 @@
 "use strict";
 
 const fs = require("fs");
-const debug = require("debug")("pdf-service");
 
 module.exports = {
   ping(req, res) {
     res.send("PONG");
   },
   sendFile(req, res) {
-    debug("send file");
+    req.app.U.logger.debug("in sendFile");
     const readStream = fs.createReadStream(`${req.temp.name}/output.pdf`);
+    let responseSent = false;
+    setTimeout(() => {
+      if (!responseSent) {
+        req.app.U.logger.error("PDF not sent");
```

Comment:
Do we want to `req.temp.removeCallback()` here? This scenario is that the pdf generation succeeded but the request that triggered the generation has gone away, probably can delete the temp files?